### ISSUE TITLE
Output from tests

### ISSUE THREAD

**Issue by bastorer** [pnsaevik/effluent#35]
Are the tests in the `/tests` directory expected to output anything? If I run, e.g. `python test_solver.py`, it runs happily with no output. 

If those are tests that intended for the user to verify installation, I'd recommend adding (at least) two print statements to each. One that prints at the beginning, saying that the test is starting, and one that prints at the end, saying if the test passed or failed.

Also, if you do decide to include a conda `.yml` file for building an Effluent environment, you'll also want to include `pytest`, which is necessary for these scripts to run.

[This is part of JOSS review https://github.com/https://github.com/openjournals/joss-reviews/issues/5554]

**Comment by pnsaevik** [comment]
There are probably many ways to do these things. In `effluent`, I have followed the fairly standard approach of separating the test suite from the main package, so regular users don't have access to the tests. Therefore, pytest is not a part of the package requirements. To set up a development environment one should clone the repo using git, and install pytest in addition to the package itself. To run the tests, cd into the repo dir and run `pytest` with no arguments. The tests should then run (54 of them). The command `python test_solver.py` does not actually run anything.

In other words, something like this:
```
cd name_of_dev_dir
git clone https://github.com/pnsaevik/effluent .
conda create -n effluent python xarray netcdf4 holoviews scipy dask
conda activate effluent
pip install pytest
pip install -e .
pytest
```

The conda environment is optional, but may be helpful on Windows. See also the "contribute code" entry from the documentation at https://github.com/pnsaevik/effluent/blob/main/docs/source/contributing.rst
